src/main/resources/application.properties

```diff
@@ -32,4 +32,3 @@ azure.cosmosdb.key=
 azure.cosmosdb.database=
 azure.cosmosdb.uri=
```

Comment:
Can we remove these in application properties?

Reply:
Yes, we can after the MIKV changes (#92) are in and I have checked in code for #93 I will do this as part of #93.